digdag-core/src/main/java/io/digdag/core/log/LogFiles.java

```diff
@@ -27,6 +28,15 @@ private LogFiles()
     private static DateTimeFormatter SESSION_TIME_FORMATTER =
         DateTimeFormatter.ofPattern("yyyyMMdd'T'HHmmssxx", ENGLISH);
 
+    private static String formatFileNameWithSuffix(String taskName, Instant firstLogTime, String agentId, String suffix) {
```

Comment:
The codes in this project follow [airlift code style](https://github.com/airlift/codestyle). 
Therefore, this `function` need to add a new line.

``` java
    private static String formatFileNameWithSuffix(String taskName, Instant firstLogTime, String agentId, String suffix)
    {
    …
    }
```

Reply:
Good catch!

@alu Could you address it?

Reply:
Thanks, i did apply the style!